tests/eko/io/test_struct.py

```diff
@@ -195,3 +196,22 @@ def test_load_opened(self, tmp_path: pathlib.Path, eko_factory: EKOFactory):
         read_opened = EKO.read(tmp_path, extract=False)
 
         assert read_closed.metadata == read_opened.metadata
+
+    def test_version(self, tmp_path: pathlib.Path, eko_factory: EKOFactory):
+        """Test asserted version. Should either be supported version, or have a postrelease addition"""
+        eko = eko_factory.get()
+        eko.close()
+        eko_factory.cache = None
+        assert eko.access.path is not None
+        read_closed = EKO.read(eko.access.path, dest=tmp_path)
+
+        version = parse(read_closed.metadata.version)
+        assert (
+            version.major + version.minor + version.micro >= 0
```

Comment:
```suggestion
            version.major + version.minor + version.micro > 0
```
no? we don't want 0.0.0

Reply:
we can even make it stricter: already major+minor has to be > 0